data_structures/linked_list/QueueLinkedList.c

```diff
@@ -0,0 +1,133 @@
+#include<stdio.h>
+#include<stdlib.h>
+
+typedef struct queue
+{
+    int data;
+    struct queue *next;
+}queue;
```

Comment:
Please maintain spacing between chunks of code. It is important to have the code well structured.